### PR TITLE
Clarify that int callbacks won't cause messages to go to the error queue

### PR DESCRIPTION
**Pull request by andreasohlund**
If there is no callback waiting when they arrive

### CODE REVIEW

**Comment by Niklas-Peter** [comment]
There is a typo "reponse" and I would add a note about how this situation is handled. As far as I understood your planned solution, you want to log a warning.

**Comment by andreasohlund** [comment]
@Niklas-Peter typo fixed, thanks.

I've added a cheecklist item in https://github.com/Particular/NServiceBus.Callbacks/issues/60 to update this doc accordingly once that is done.

@Particular/docs-maintainers ready for review

**Comment by SzymonPobiega** [comment]
@andreasohlund can you check the build error?

File: nservicebus/messaging/callbacks.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ The callback functionality can be split into three categories based on the type
 
 The integer response scenario allows any integer value to be returned in a strong typed manner.
 
+WARN: Note that this type of callback won't cause response messages to end up in the [error queue](/nservicebus/recoverability) if no callback is registered since there there is no way those messages can be recovered.
```
Comment (SimonCropp) [inline nservicebus/messaging/callbacks.md]: Warning. Not warn

Comment (andreasohlund) [inline nservicebus/messaging/callbacks.md]: duh, fixed!

**Pull request merged**